internal/v5/search.go

```diff
@@ -21,30 +22,30 @@ const maxConcurrency = 20
 // GET search[?text=text][&autocomplete=1][&filter=value…][&limit=limit][&include=meta][&skip=count][&sort=field[+dir]]
 // https://github.com/juju/charmstore/blob/v4/docs/API.md#get-search
 func (h *ReqHandler) serveSearch(_ http.Header, req *http.Request) (interface{}, error) {
```

Comment:
An alternative to duplicating lots of code might just be to add a "expandMultiSeries" arg to this function,
although I guess that's at odds to keeping the main version as clean as possible.